src/python/pants/notes/master.rst

```diff
@@ -4,6 +4,68 @@ Master Pre-Releases
 This document describes ``dev`` releases which occur weekly from master, and which do
 not undergo the vetting associated with ``stable`` releases.
 
+1.3.0dev3 (12/02/2016)
```

Comment:
this should probably match `version.py`, which now reflects the canonicalized version string @ `1.3.0.dev3` (and should hopefully be the last change we have to make around this).